src/main/java/me/eccentric_nz/TARDIS/database/ResultSetTraveledTo.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (C) 2020 eccentric_nz
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package me.eccentric_nz.TARDIS.database;
+
+import me.eccentric_nz.TARDIS.TARDIS;
+
+import java.sql.*;
+
+import org.bukkit.World.Environment;
+
+import org.bukkit.entity.Player;
```

Comment:
```suggestion
import me.eccentric_nz.TARDIS.TARDIS;
import org.bukkit.World.Environment;
import org.bukkit.entity.Player;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.ResultSet;
import java.sql.SQLException;
```

IntelliJ suggested optimized imports